scripts/migration/migrate_registered_meta.py

```diff
@@ -0,0 +1,80 @@
+""" Changes existing registered_meta on a node to new schema layout
+required for the prereg-prize
+"""
+import re
+import json
+import sys
+import logging
+
+from modularodm import Q
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+IMMUTABLE_KEYS = frozenset(['datacompletion', 'summary'])
+
+def get_registered_nodes():
+    return models.Node.find(
+        Q('is_registration', 'eq', True)
```

Comment:
We would only need those registrations that have not been migrated. However, a flag on the Node class for one migration seems like overkill. Any suggestion on how to keep track of it?